mappings/net/minecraft/client/render/debug/PathfindingDebugRenderer.mapping

```diff
@@ -2,6 +2,29 @@ CLASS net/minecraft/class_868 net/minecraft/client/render/debug/PathfindingDebug
 	FIELD field_4614 client Lnet/minecraft/class_310;
 	FIELD field_4615 pathTimes Ljava/util/Map;
 	FIELD field_4616 paths Ljava/util/Map;
+	FIELD field_4617 pathSizes Ljava/util/Map;
+	METHOD <init> (Lnet/minecraft/class_310;)V
+		ARG 1 client
+	METHOD method_20554 getDistance (Lnet/minecraft/class_4184;Lnet/minecraft/class_2338;)F
+		ARG 0 camera
+		ARG 1 pos
+	METHOD method_20555 drawPath (Lnet/minecraft/class_4184;Lnet/minecraft/class_11;)V
+		ARG 0 camera
+		ARG 1 path
+	METHOD method_20556 renderPath (Lnet/minecraft/class_4184;Lnet/minecraft/class_11;FZZ)V
```

Comment:
`renderPath` will set up a GL context around `drawPath` and will also add things like labels, boxes and other things through `renderPathStruts`.

`drawPath` literally just draws a path using the Tesselator. I didn't want to override them because they seemed to act very differently.